modules/ROOT/pages/shared-resources.adoc

```diff
@@ -208,41 +208,47 @@ By default, when you deploy either an application associated with a domain or a
 [NOTE]
 Mule domain projects cannot be installed using Runtime Manager.
 
-To set this in Studio, open the drop-down menu next to the play button and select *Run Configurations*.
+=== Deploy from Anypoint Studio
 
-image::run-configurations-1.png[run+configurations+1]
+The following steps describe how to deploy your domain project and the associated applications from Studio:
 
-Then pick the *General* tab, and tick or untick the boxes next to the projects that you want to always deploy together with the application that is currently selected on the navigation menu to the right.
+. Open the drop-down menu next to the play button and select *Run Configurations*.
++
+image::mruntime-run-configuration.png[run+configuration]
++
+. Select the *General* tab, and tick or untick the boxes next to the projects that you want to always deploy together with the application that is currently selected on the navigation menu to the right.
```

Comment:
```suggestion
. Select the *General* tab and then select the boxes next to the projects that you want to always deploy together with the application that is currently selected in the navigation menu to the right.
```
You would not unselect to always deploy together.... I think mentioning only selecting is clear, but if you like, you can add a phrase about unselecting the projects that you want to omit from the group that is always deployed together.